common/src/gas_provider/node.rs

```diff
@@ -245,8 +249,8 @@ impl<ExternalId: Clone, Id: Clone + Copy, Balance: Zero + Clone + Copy>
             Self::External { lock, .. }
```

Comment:
`pub fn lock(&self) -> Balance` instead of option now

Reply:
lock_mut also

Reply:
done